samples/mini-games/settings.gradle.kts

```diff
@@ -1,15 +1,18 @@
-// TODO: remove this once we have publishing setup for all artifacts
-includeBuild("../../") {
-    dependencySubstitution {
-        substitute(module("com.utopia-rise:godot-gradle-plugin")).with(project(":godot-gradle-plugin"))
+// without this doing:
```

Comment:
this is sadly required so that we can use the `plugins` block.

Reply:
I would also add a todo here to remove it like you have done below

Reply:
yep, I'll add it to the samples README though. We still need some documentation on how to try a local build.